test/integration/apiserver/field_validation_test.go

```diff
@@ -2647,16 +2638,11 @@ func TestFieldValidationApplyCreateCRDSchemaless(t *testing.T) {
 	}
 }
 
-// TestFieldValidationApplyUpdateCRD tests apply patch requests containing duplicate fields
+// testFieldValidationApplyUpdateCRD tests apply patch requests containing duplicate fields
 // on existing objects, for CRDs with schemas
 // Note that even prior to server-side validation, unknown fields were treated as
 // errors in apply-patch and are not tested here.
-func TestFieldValidationApplyUpdateCRD(t *testing.T) {
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ServerSideFieldValidation, true)()
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ServerSideApply, true)()
```

Comment:
this feature gate is removed,  is no longer needed?

Reply:
Yeah, GA